.github/workflows/build-push-images.yaml

```diff
@@ -124,7 +122,7 @@ jobs:
           containerfiles: |
             ./images/kcp-registrar/Dockerfile
           build-args: |
-            KCP_BRANCH=release-0.7
+            KCP_BRANCH=v0.8.0
```

Comment:
latest is [v0.8.2](https://github.com/kcp-dev/kcp/releases/tag/v0.8.2)
v0.8.0 did not reach stable because of issues. We should start with a later version.

Reply:
As discussed, next PR will do automatic upgrade to v0.8.2 (or whatever is the latest one available at the time)